app/src/main/java/uk/gov/onelogin/optin/ui/OptInScreen.kt

```diff
@@ -0,0 +1,126 @@
+package uk.gov.onelogin.optin.ui
+
+import android.net.Uri
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.rememberScrollState
+import androidx.compose.foundation.verticalScroll
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.collectAsState
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.platform.LocalUriHandler
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.semantics.heading
+import androidx.compose.ui.semantics.semantics
+import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.tooling.preview.PreviewFontScale
+import androidx.compose.ui.tooling.preview.PreviewLightDark
+import androidx.compose.ui.tooling.preview.PreviewScreenSizes
+import androidx.hilt.navigation.compose.hiltViewModel
+import uk.gov.android.onelogin.R
+import uk.gov.android.ui.theme.m3.GdsTheme
+import uk.gov.android.ui.theme.mediumPadding
+import uk.gov.android.ui.theme.smallPadding
+import uk.gov.android.ui.theme.xsmallPadding
+
+
+@Composable
+fun OptInScreen(
+    viewModel: OptInViewModel = hiltViewModel<OptInViewModel>(),
+    onComplete: () -> Unit,
+) {
+    val uriHandler = LocalUriHandler.current
+    val state = viewModel.uiState.collectAsState(OptInUIState.PreChoice)
+    GdsTheme {
```

Comment:
KtLint didn't accept it, so it had to be changed. It thought the import for it was unused.

Reply:
I was responding to the wrong thing, apologies. I can't remember why I did this. I think the Theme wasn't applying otherwise. I will double check

Reply:
![WithoutGDSThemeSet](https://github.com/user-attachments/assets/ff8a9583-fc53-48fe-94bd-6324c53d68de)
This is what it looks like when I remove the GDSTheme from that section. Though I agree that it should just be set once and not all over the place or for each screen.

I can obviously set GDSTheme in the Activities setContent block, but I'd rather not have to test the whole app to make sure this doesn't break something else.

Reply:
It is because I am using the material 3 Theme and Composables, so I am leaving it in place for now and we will have to decide what we will use going forward.